tests/zfs-tests/tests/unit/libtest/Makefile.am

```diff
@@ -0,0 +1,5 @@
+pkgdatadir = $(datadir)/@PACKAGE@/zfs-tests/tests/unit/libtest
+dist_pkgdata_SCRIPTS = \
+	libtest_001_pos.ksh \
```

Comment:
Could we rename these to have meaningful filenames describing what's tested?

Reply:
it is testing libtest.shlib, no?  or do you mean in finer detail, such as libtest_vdev_001_pos.ksh? or perhaps libtest.shlib_001_pos.ksh?

Reply:
I'm thinking something more like check_pool_status.ksh vdev_tests.ksh and maybe utils_tests.ksh. Granted they all test libtest.shlib, but the contents of each file is a logical group of similar tests. With more descriptive names, when future authors want to know if there's a file they can add a unit test to as opposed to creating a new one descriptive names will be more helpful than libtest_0*.ksh.

Reply:
good idea, I never liked the numbering anyway :-)